HabboHotel/Users/Ignores/IIgnoresManager.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Plus.HabboHotel.Users.Ignores
+{
+    public interface IIgnoresManager
+    {
+       bool Init(Habbo player);
+       bool TryGet(Habbo userId);
+       bool TryAdd(Habbo userId);
+       bool TryRemove(Habbo userId);
+       ICollection<int> IgnoredUserIds();
+       void Dispose();
+       IReadOnlyCollection<string> GetIgnoredUsers(Habbo uid);
```

Comment:
Parameter type is `Habbo` but variable name is called `uid`. I would suggest only to pass the required data needed for the query, `int userId` should suffice.